client/src/apps/code-collabo/components/AppPageStructure.tsx

```diff
@@ -28,27 +35,48 @@ export default function PageStructure({ children }: { children: ReactNode }) {
       <style jsx global>{ types }</style>
       <style jsx global>{ main }</style>
 
-      {/* Sidebar */}
-      <div>
-        <div><b>CODE COLLABO Sidebar Logo</b> goes here</div>
-        <nav>
-          <span><b>Sidebar Nav Menu</b> goes here: </span>
-          <button>Overview</button>
-          <button>Projects</button>
-          <button>Careers</button>
-          <button>Donate</button>
-        </nav>
-      </div>
-
-      {/* Page Content */}
-      <div>
-        <header>
-          <h2 className='page-title'>{thisPage}</h2>
-        </header>
-        <main>
-          { children }
-        </main>
-      </div>
+      { !isMobile ? (
+        // Web
+        <div className='container'>
+          {/* Sidebar */}
+          <div className='sidebar'>
+            <div><b>CODE COLLABO Sidebar Logo</b> goes here</div>
+            <nav>
+              <span><b>Sidebar Nav Menu</b> goes here: </span>
+              <Link href='/code-collabo'>Overview</Link>
+              <Link href='/code-collabo/projects'>Projects</Link>
+              <Link href='/code-collabo/careers'>Careers</Link>
+              <Link href='/code-collabo/donate'>Donate</Link>
+            </nav>
+          </div>
+
+          {/* Page Content */}
+          <div className='content'>
+            <header>
+              <h2 className='page-title'>{thisPage}</h2>
+            </header>
+            <main>
+              { children }
+            </main>
+          </div>
+        </div>
+      ) : (
+        // mobile
+        <div className='container-mobile'>
+          <header className='header-mobile'>
+            <div className='icon-outline'>
+              <Image src='/code-collabo/hamburger.png' alt='hamburger-icon' width={18} height={11}/>
+            </div>
+            <h2 className='page-title-mobile'>{thisPage}</h2>
+            <div className='icon-outline'>
+              <Image src='/code-collabo/menu.png' alt='hamburger-icon' width={18} height={11}/>
+            </div>
+          </header>
+          <main>
+            { children }
+          </main>
+        </div>
```

Comment:
For example: One html code (with "split" CSS styles) can take care of the visual aspect of the UI for both desktop & mobile. The javascript `isMobile` check can do well for triggering functionality.
> Just an example. But see how nav & maybe modals are handled in this project: https://github.com/Ifycode/firebase-auth-guide-app (see both their CSS and JS files).
> Although the example uses plain css and js unlike this project, it's just the idea of that separation of concern we need from there. And to prevent the possibility of unnecessary UI flickering in the future or content jumping around in funny ways due to using js for what CSS would have done a bit more cleanly.

Reply:
Don't worry, I'm not expecting everything I'm explaining/reviewing to be clear at the same time. We'll discuss how to tackle it in the call @Akpjunior94

Reply:
- Fix in #85